dash/include/dash/memory/GlobHeapMem.h

```diff
@@ -429,6 +429,23 @@ class GlobHeapMem
     return unit_local_size;
   }
 
```

Comment:
Ah, yes. Actually I want to get rid of the division ops somehow and multiply the byte-based offsets instead. As we know from ParComp, division is horribly expensive and pointer increments should have as little overhead as possible.